pkg/common/crypto/default.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
```

Comment:
i do not think this package makes sense.

- the default packages are held under framework.
- we shouldn’t need a second wrapper of tink.

Reply:
this Package includes common crypto operations to be used by the framework, the crypto under `provider` are for specific operations like encrypt/decrypt or sign/verify and are used to build an instance of this package.

Reply:
as discussed, I'll remove this package and add provider context option for Crypto in the Aries Framework.

Reply:
removed